examples/stm32/f1/other/usb_hid/usbhid.c

```diff
@@ -279,6 +279,13 @@ int main(void)
 	rcc_clock_setup_in_hsi_out_48mhz();
 
 	rcc_periph_clock_enable(RCC_GPIOA);
+	/* Hack to force renumeration */
+	gpio_set_mode(GPIOA, GPIO_MODE_OUTPUT_2_MHZ,
+		GPIO_CNF_OUTPUT_PUSHPULL, GPIO12);
+	gpio_clear(GPIOA, GPIO12);
+	for (unsigned i = 0; i < 800000; i++) {
```

Comment:
yep, it's pretty arbitrary how long you need to bounce for to trigger reenumeation.  we've got plenty of "hacky delays" like this in examples